app/views/middleware_server/_add_jdbc_driver.html.haml

```diff
@@ -30,6 +30,26 @@
                        :locals => {:object_name => "jdbc_driver", :method => "file", :custom_options => {"ng-required" => "true", "ng-model" => "jdbcDriverModel.filePath", "require-file" => "handle"}}
 
           .form-group
+            %label.col-md-4.control-label{:for => "xa_driver_ds_cb"}
+              = _("XA Datasource:")
+            .col-md-8
+              %input#xa_driver_ds_cb{:type      => "checkbox",
+                              "bs-switch"       => "",
+                              "switch-on-text"  => _("Yes"),
+                              "switch-off-text" => _("No"),
+                              "switch-change"   => "onDriverXaChange()",
+                              "ng-model"        => "jdbcDriverModel.xaDatasource"}
+
+          .form-group
+            %label.col-md-4.control-label{:for => "choose_datasource_input"}
+              = _("Datasource:")
+            .col-md-8
+              %select.form-control#chooose_driver_ds{"name"        => "choose_driver_ds_input",
```

Comment:
this looks like a comment - is this a github issue? (the choose driver part above)